### PR TITLE
core-services/prow/02_config/generate-boskos: For spitting out lots of names

### PR DESCRIPTION
**Pull request by wking**
Quotas for some providers depend on both account-scoped resources (e.g. [AWS elastic IPs default to five per account][1]), region-scoped resources (e.g. [AWS VPCs default to five per region][1]), potentialy other scoping (e.g. [AWS NAT gateways default to five per availability zone][1]).  This commit moves us away from account-scoped leases and towards region-scoped leases in account-scoped buckets.  That allows us to say things like "on Azure, we have more capacity in centralus than we do in our other regions".

The generating script is because typing out `us-east-1` 50 times is tedious and hard to review, and today there is apparently no support in the Boskos config directly to ask for $N copies of a given name.

I'm also adding myself as an owner, so DPTP doesn't have to bother with lease adjustment.  Other CI-platform maintainers should feel free to add themselves as well, if they want to approve lease adjustments to their platforms.

[1]: https://docs.openshift.com/container-platform/4.5/installing/installing_aws/installing-aws-account.html#installation-aws-limits_installing-aws-account

### CODE REVIEW

File: core-services/prow/02_config/generate-boskos.py
Comment (stevekuznetsov) [inline core-services/prow/02_config/generate-boskos.py:58]:
```python
     CONFIG['openstack-ppc64le-quota-slice']['openstack-ppc64le-{0:0>2}'.format(i)] = 1
```

File: core-services/prow/02_config/generate-boskos.py
Comment (stevekuznetsov) [inline core-services/prow/02_config/generate-boskos.py:61]: similar?

File: core-services/prow/02_config/generate-boskos.py
Comment (wking) [reply core-services/prow/02_config/generate-boskos.py]: Fixed with 204dc967ab -> b831dd7a56.

File: core-services/prow/02_config/generate-boskos.py
Comment (wking) [reply core-services/prow/02_config/generate-boskos.py]: Pivoted to `range(10, 18)` with 204dc967ab -> b831dd7a56.

**Comment by wking** [comment]
I've pushed  cc5e7eb -> e303bcd070, rebasing around #11772 and adding `make check-boskos`, `make boskos-config`, and a `boskos-config` presubmit.  Which seems to be [failing][1]...

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/11752/rehearse-11752-pull-ci-openshift-release-master-boskos-config/1305996033737625600

**Comment by wking** [comment]
Fixed the position of the new presubmit job in the YAML file with e303bcd070 -> 0cf4ca96f8.  Just have to figure out which image to use for the presubmit so we get `make`, or whether we want to skip `make` and just inline the command it wraps.

**Comment by wking** [comment]
I've pushed 0cf4ca96f8 -> 2dd18757cf, dropping `make` and using the `registry.svc.ci.openshift.org/ci/python-validation` image, at @stevekuznetsov 's suggestion.

**Review by petr-muller — APPROVED** [review]
LGTM but this needs the other PR to make the underlying image have `yaml` so holding

/hold

**Comment by wking** [comment]
With 2dd18757cf -> 5cf5ba3f87, I've rebased onto master, claimed `python3` because openshift/release#11869 is still unsettled, and limited this change to a no-op reshuffling.  I'll come back with follow-up work after this PR lands to actually move AWS/GCP/Azure to per-region leases.

File: hack/validate-boskos.sh
Comment (wking) [inline hack/validate-boskos.sh:14]: [Grr][1]:

```
hack/validate-boskos.sh: line 14: git: command not found
```

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/11752/rehearse-11752-pull-ci-openshift-release-master-boskos-config/1314310508639162368

Comment (wking) [reply hack/validate-boskos.sh]: I've pushed 5cf5ba3f87 -> 204f9a738a to drop the `git` dependency.

**Comment by wking** [comment]
Me [giving up][1] on getting `python` to point to a Python with PyYAML means we can drop [the hold][2].

/hold cancel

[1]: https://github.com/openshift/release/pull/11752#issuecomment-705821737
[2]: https://github.com/openshift/release/pull/11752#pullrequestreview-489610418

**Review by stevekuznetsov — APPROVED** [review]

File: core-services/prow/02_config/generate-boskos.py
Comment (coverprice) [inline core-services/prow/02_config/generate-boskos.py:17]: code golf:
```
'libvirt-ppc64le-quota-slice': {
  f'libvirt-ppc64le-{i}-{j}': 1
  for j in range(4)
  for in in range(2)
},
```

**Comment by wking** [comment]
Followed up for per-region Azure leases in #12584.

File: core-services/prow/02_config/generate-boskos.py
Comment (wking) [reply core-services/prow/02_config/generate-boskos.py:17]: Fine with me.  File a PR?

**Pull request merged**
